Modules/Sources/Yosemite/Model/Mocks/ActionHandlers/MockAppSettingsActionHandler.swift

```
@@ -57,8 +57,11 @@ struct MockAppSettingsActionHandler: MockActionHandler {
         case .upsertProductsSettings(_, _, _, _, _, _, _, let onCompletion):
             onCompletion(nil)
         case .resetEligibilityErrorInfo,
-                .setTelemetryAvailability
-            :
+                .setTelemetryAvailability,
+                .getAppPasswordsExperimentSettingState,
+                .getPOSSurveyCurrentMerchantNotificationScheduled,
+                .getPOSSurveyPotentialMerchantNotificationScheduled,
+                .getHasPOSBeenOpenedAtLeastOnce:
             break
         default: unimplementedAction(action: action)
```

Hmm. Should we actually have this default, if this is causing issues?

---

I'm in the middle about it: 

I think we shouldn't use `default` in general to just hide from the compiler that the action hasn't been handled, despite being a mock it just forces a runtime error for no reason imho.

On the other side is not causing much trouble since only runs when we have to trigger the screenshot tests. I had to add a couple more actions here on a separate PR while working with WooCommerceScreenshots but that's about it. Updating it now means adding 67 actions to the switch statement that are missing 😅